sqlalchemy_redshift/commands.py

```diff
@@ -167,6 +172,11 @@ def __init__(self, select, unload_location, access_key_id=None,
                 '"delimiter" parameter must be a single character'
             )
 
+        if header is not None and fixed_width is not None:
```

Comment:
Should use `if header and fixed_width is not None` since fixed_width should be fine to use if header is False.